dallinger/data.py

```diff
@@ -310,7 +317,7 @@ def ingest_to_model(file, model, engine=None):
     postgres_copy.copy_from(
         file, model, engine, columns=columns, format="csv", HEADER=False
     )
-    fix_autoincrement(model.__table__.name)
+    fix_autoincrement(engine, model.__table__.name)
```

Comment:
It seems like this should be handling the `engine=None` default case in some way here.

Reply:
At this point `engine` will have been bound to `db.engine` in the default case, no?